<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MountAngleCalculationClass</class>
 <widget class="QMainWindow" name="MountAngleCalculationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MountAngleCalculation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="fileformat_cmb">
        <item>
         <property name="text">
          <string>OpenRTK330LI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RTK330LA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>INS401</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filepath_edt"/>
      </item>
      <item>
       <widget class="QToolButton" name="select_btn">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget_config">
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>priLeverArm</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>rotationRBV</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>odoLeverarm</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>userLeverArm</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Z</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="time_lb">
          <property name="text">
           <string>00:00:00:000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_decode">
          <property name="text">
           <string>decode</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>ins result</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="process_filepath_edt">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="process_select_btn">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_split">
        <property name="text">
         <string>split</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>time slices</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="time_slices_comb">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>startime</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="starttime_edt"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>endtime</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="endtime_edt">
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>result</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="result_edt">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_calculate_all">
        <property name="text">
         <string>calculate all slices</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_calculate">
        <property name="text">
         <string>calculate</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MountAngleCalculation.qrc"/>
 </resources>
 <connections/>
</ui>
